NitroxServer/ServerLogger.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.IO;
+
+namespace NitroxServer
+{
+    class ServerLogger : TextWriter
```

Comment:
Does the Log class not handle writing to console?  I forget

Reply:
Yup it does. `ServerLogger` should be removed and use `Log` instead.

Reply:
The log class writes to console however it doesnt write console to a file hence the addition of ServerLogger which writes to both a file and console

Reply:
Maybe enhance or inherit from `Log` and add an additional file writing option instead? Or should we opt for a third-party logger that does it for us?